2021-10-06 22:15:38,071 INFO    MainThread:34014 [wandb_setup.py:_flush():69] setting env: {}
2021-10-06 22:15:38,071 INFO    MainThread:34014 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-06 22:15:38,071 INFO    MainThread:34014 [wandb_init.py:_log_setup():357] Logging user logs to /Users/kiranganeshan/reading-group/RL/wandb/run-20211006_221538-3sreffb6/logs/debug.log
2021-10-06 22:15:38,071 INFO    MainThread:34014 [wandb_init.py:_log_setup():358] Logging internal logs to /Users/kiranganeshan/reading-group/RL/wandb/run-20211006_221538-3sreffb6/logs/debug-internal.log
2021-10-06 22:15:38,072 INFO    MainThread:34014 [wandb_init.py:init():390] calling init triggers
2021-10-06 22:15:38,072 INFO    MainThread:34014 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-10-06 22:15:38,072 INFO    MainThread:34014 [wandb_init.py:init():435] starting backend
2021-10-06 22:15:38,072 INFO    MainThread:34014 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-06 22:15:38,088 INFO    MainThread:34014 [backend.py:ensure_launched():135] starting backend process...
2021-10-06 22:15:38,096 INFO    MainThread:34014 [backend.py:ensure_launched():140] started backend process with pid: 34041
2021-10-06 22:15:38,100 INFO    MainThread:34014 [wandb_init.py:init():440] backend started and connected
2021-10-06 22:15:38,108 INFO    MainThread:34014 [wandb_init.py:init():499] updated telemetry
2021-10-06 22:15:38,109 INFO    MainThread:34014 [wandb_init.py:init():522] communicating current version
2021-10-06 22:15:38,813 INFO    MainThread:34014 [wandb_init.py:init():527] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-06 22:15:38,814 INFO    MainThread:34014 [wandb_init.py:init():535] communicating run to backend with 30 second timeout
2021-10-06 22:15:38,974 INFO    MainThread:34014 [wandb_init.py:init():562] starting run threads in backend
2021-10-06 22:15:41,653 INFO    MainThread:34014 [wandb_run.py:_console_start():1651] atexit reg
2021-10-06 22:15:41,654 INFO    MainThread:34014 [wandb_run.py:_redirect():1525] redirect: SettingsConsole.REDIRECT
2021-10-06 22:15:41,655 INFO    MainThread:34014 [wandb_run.py:_redirect():1530] Redirecting console.
2021-10-06 22:15:41,658 INFO    MainThread:34014 [wandb_run.py:_redirect():1586] Redirects installed.
2021-10-06 22:15:41,658 INFO    MainThread:34014 [wandb_init.py:init():587] run started, returning control to user process
2021-10-06 22:15:41,682 INFO    MainThread:34014 [wandb_watch.py:watch():43] Watching
2021-10-06 22:15:41,683 INFO    MainThread:34014 [wandb_run.py:_atexit_cleanup():1621] got exitcode: 1
2021-10-06 22:15:41,684 INFO    MainThread:34014 [wandb_run.py:_restore():1593] restore
2021-10-06 22:15:44,175 INFO    MainThread:34014 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 689
}

2021-10-06 22:15:44,372 INFO    MainThread:34014 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 689
}

2021-10-06 22:15:45,002 INFO    MainThread:34014 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 3025
}

2021-10-06 22:15:45,107 INFO    MainThread:34014 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 5592
}

2021-10-06 22:15:45,212 INFO    MainThread:34014 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 5592
}

2021-10-06 22:15:45,314 INFO    MainThread:34014 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5592
  total_bytes: 5592
}

2021-10-06 22:15:45,420 INFO    MainThread:34014 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5592
  total_bytes: 5592
}

2021-10-06 22:15:45,521 INFO    MainThread:34014 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5592
  total_bytes: 5592
}

2021-10-06 22:15:45,623 INFO    MainThread:34014 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5592
  total_bytes: 5592
}

2021-10-06 22:15:45,733 INFO    MainThread:34014 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5592
  total_bytes: 5592
}

2021-10-06 22:15:46,074 INFO    MainThread:34014 [wandb_run.py:_wait_for_finish():1746] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5592
  total_bytes: 5592
}
local_info {
}

2021-10-06 22:15:47,193 INFO    MainThread:34014 [wandb_run.py:_append_files():2009] logging synced files
